app.js

```diff
@@ -8,10 +8,14 @@ const path       = require('path')
 const app        = express()
 
 app.use(bodyParser.urlencoded({extended: true}))
+app.use(bodyParser.json({ limit: "50mb" }));
 app.use(express.static('reports'));
 
+const apikey = 'PUT YOUR API KEY HERE'

```

Comment:
Not sure I like this - and not if there potential for it to be hardcoded and stored in some public repo. Can this be passed in via a system environment variable etc?

Reply:
you are right.  I added a new commit to handle api key from environment file

Reply:
Nice one. At least that way it can be excluded from the repo easier :)